### PR TITLE
gping: 1.2.6 -> 1.2.7

### PR DESCRIPTION
**Pull request by iAmMrinal0**
###### Motivation for this change
Package update: [Tag v1.2.7](https://github.com/orf/gping/releases/tag/gping-v1.2.7)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by trofi** [comment]
Result of `nixpkgs-review pr 156969` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gping</li>
  </ul>
</details>

**Review by trofi — APPROVED** [review]

File: pkgs/tools/networking/gping/default.nix [hunk 1]
```diff
@@ -8,26 +8,24 @@
...
     repo = "gping";
     rev = "gping-v${version}";
-    sha256 = "sha256-Sxmwuf+iTBTlpfMFCEUp6JyEaoHgmLIKB/gws2KY/xc=";
+    sha256 = "sha256-7o7Tj0jWFIOLmpHXWT6zcyowm7vnqMDTf0S4zHkWQ2Q=";
   };
 
-  cargoSha256 = "sha256-xEASs6r5zxYJXS+at6aX5n0whGp5qwuNwq6Jh0GM+/4=";
-
-  patches = [
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/gping/default.nix:23]: Please remove the unused fetchpatch from inputs.

**Comment by vcunat** [comment]
If OfBorg is right, this PR would break the build on darwin: https://hydra.nixos.org/job/nixpkgs/trunk/gping.x86_64-darwin

**Comment by iAmMrinal0** [comment]
> If OfBorg is right, this PR would break the build on darwin: [hydra.nixos.org/job/nixpkgs/trunk/gping.x86_64-darwin](https://hydra.nixos.org/job/nixpkgs/trunk/gping.x86_64-darwin)

Thanks for the heads-up, @vcunat, I had missed checking the failures. I have updated the PR and looks all green.

**Pull request merged**